packages/editor/web/src/RichContentEditor/InnerRCE.jsx

```diff
@@ -6,19 +6,33 @@ import RichContentEditor from './RichContentEditor';
 import styles from '../../statics/styles/rich-content-editor.scss';
 import 'wix-rich-content-common/dist/statics/styles/draftDefault.rtlignore.scss';
 import { __convertToRawWithoutVersion } from '../../lib/editorStateConversion';
+import { LINK_PREVIEW_TYPE } from 'wix-rich-content-common';
 import { cloneDeep } from 'lodash';
 
 class InnerRCE extends Component {
   constructor(props) {
     super(props);
     const { innerRCERenderedIn, config, editorState } = props;
-    this.config = this.removeAnchorFromLink(cloneDeep(config));
+    this.config = this.cleanConfig(cloneDeep(config));
     this.plugins = config[innerRCERenderedIn].innerRCEPlugins;
     this.state = {
       editorState,
     };
   }
 
+  cleanConfig = config => {
+    const configWithoutAnchor = this.removeAnchorFromLink(config);
+    const configWithoutLinkPreview = this.removeLinkPreview(configWithoutAnchor);
+    return configWithoutLinkPreview;
+  };
```

Comment:
in my opinion it's a bit more clear
```suggestion
  cleanConfig = config => {
    let clearConfig = config;
    clearConfig = this.removeAnchorFromLink(clearConfig);
    clearConfig = this.removeLinkPreview(clearConfig);
    return clearConfig;
  };
```

Reply:
Sure.